README.md

```diff
@@ -45,3 +45,15 @@ Finally use for build
 Known issues:
 Gulp watch is not interupable so use `docker stop <hash>` from other console
 Gulp watch will not reload browser in this case cause of ports issue(localhost required).
+
+
+For old projects managed with ruby and gems use tag omega_4_3 (for compass 0.12) and omega_4_4 (for compass 1+)̨.
+For 4_3:
+``` bash
+docker build -t omega_4_3 omega_4_3
+alias omega_4_3='docker run --rm -it -v $(pwd):/work omega_4_3 '
```

Comment:
maybe, but sometimes in one day you may have several projects versions in one day. in this case would be cool to have containers ready.